Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -862,26 +862,13 @@ - (UIView *)loadViewForDownloadingAttachment
 {
     OWSAssert(self.attachmentPointer);
 
-    UIView *customView = [UIView new];
-    switch (self.attachmentPointer.state) {
-        case TSAttachmentPointerStateEnqueued:
-            customView.backgroundColor
-                = (self.isIncoming ? [UIColor ows_messageBubbleLightGrayColor] : [UIColor ows_fadedBlueColor]);
-            break;
-        case TSAttachmentPointerStateDownloading:
-            customView.backgroundColor
-                = (self.isIncoming ? [UIColor ows_messageBubbleLightGrayColor] : [UIColor ows_fadedBlueColor]);
-            break;
-        case TSAttachmentPointerStateFailed:
-            customView.backgroundColor = [UIColor grayColor];
-            break;
-    }
-
-    AttachmentPointerView *attachmentPointerView =
+    AttachmentPointerView *downloadView =
         [[AttachmentPointerView alloc] initWithAttachmentPointer:self.attachmentPointer isIncoming:self.isIncoming];
-    [customView addSubview:attachmentPointerView];
-    [attachmentPointerView autoPinWidthToSuperviewWithMargin:20.f];
-    [attachmentPointerView autoVCenterInSuperview];
+
+    UIView *wrapper = [UIView new];
+    [wrapper addSubview:downloadView];
+    [downloadView autoPinWidthToSuperview];
+    [downloadView autoVCenterInSuperview];
```

Comment:
do we need to constrain `downloadView` height?

Reply:
Done. There's places in our codebase where we skip these kinds of constraints, e.g. when our "measurement" logic guarantees there will be enough space for the centered view.  I'm never sure if these extra constraints are worth it.